### PR TITLE
pythonPackages.py3status: 3.0 -> 3.5

### PR DESCRIPTION
**Pull request by dotlambda**
I also added some patches to fix binary paths.
`pkgs.file` is needed to detect file encoding.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Just to ping the maintainer: @garbas

### CODE REVIEW

**Review by 0xABAB — APPROVED** [review]

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -9217,15 +9217,16 @@ in {
...
     src = pkgs.fetchFromGitHub {
       owner = "ultrabug";
       repo = "py3status";
       rev = version;
-      sha256 = "1mnl0rfbnimcpp7q9hva0x9jfa58j4nc27r9kvaii8869kzssw48";
+      sha256 = "1fapcq2yqc4kc2nlvg3cpypy3fiykphzjp52yjjfr2wrvr0wjyxz";
     };
-    propagatedBuildInputs = with self; [ requests ];
+    propagatedBuildInputs = with self; [ requests dbus-python pkgs.file ];
+    doCheck = false;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: Why do you disable tests?

**Comment by FRidh** [comment]
ping

**Comment by dotlambda** [comment]
Sorry for not getting back to you, exams and stuff...

I actually disabled tests to make this work on Python 3, but I think I'll need to work on this again to also call the python3 version in all-packages.nix.
Then, I'll also try to run the tests in python3 (I think they are simply not run when using version 2), but because of all the modules used by some component of py3status, I don't think I will have much luck. NixOS and dynamic loading of dependencies just don't go together.

**Comment by dotlambda** [comment]
Actually, I think I'll open a new pull request when I have the time.

**Pull request closed without merging**